deltadb/src/deltadb.c

```diff
@@ -444,13 +443,17 @@ static int log_recover( struct deltadb *db, time_t snapshot )
 	return 1;
 }
 
-struct deltadb * deltadb_create( const char *logdir )
+struct deltadb * deltadb_create( const char *logdir, time_t timestamp )
```

Comment:
Let's make the API more specific here, so as to avoid misuse:

`deltadb_create(logdir)` - creates a new instance at the current time, and permits appending.
`deltadb_create_snapshot(logdir,time)` - creates a new instance at a prior time, and inhibits appending.